bin/build_offload.sh

```diff
@@ -118,7 +117,7 @@ if [ "$1" != "nocmake" ] && [ "$1" != "install" ] ; then
    echo "Use ""$0 nocmake"" or ""$0 install"" to avoid FRESH START."
    echo rm -rf $BUILD_DIR/build/offload
    rm -rf $BUILD_DIR/build/offload
-   MYCMAKEOPTS="$COMMON_CMAKE_OPTS -DCMAKE_BUILD_TYPE=Release $AOMP_ORIGIN_RPATH"
+   MYCMAKEOPTS="$COMMON_CMAKE_OPTS -DCMAKE_PREFIX_PATH=$ROCM_CMAKECONFIG_PATH;$AOMP_INSTALL_DIR/lib/cmake -DCMAKE_BUILD_TYPE=Release $AOMP_ORIGIN_RPATH"
```

Comment:
Needs double quotes:
`"$ROCM_CMAKECONFIG_PATH;$AOMP_INSTALL_DIR/lib/cmake"`

Reply:
I think here the MYCMAKEOPTS="...  -DCMAKE_PREFIX_PATH=$ROCM_CMAKECONFIG_PATH;$AOMP_INSTALL_DIR/lib/cmake" will expand correctly.